src/exercises/Exercises.js

```diff
@@ -94,6 +96,12 @@ export default function Exercises({
       setActivityOver(true);
       exerciseNotification.unsetExerciseCounter();
       exerciseNotification.updateReactState();
+      let currentAutoPronounce = LocalStorage.getAutoPronounceInExercises();
```

Comment:
could this be named `updateAutoPronounceAtEndOfSession` and maybe moved elsewhere... although most of it is actually the test ... 

you know what i'm thinking: it might be less code (e.g. this would not be needed), and also simpler for the learner if we kept only two modes: **Off** and **On**. If the user starts a new session, and they don't want it, they simply turn it off, no biggie. And that really everybody understands. Session might be clear to me and you but not the average Frederik. What do you think?

Reply:
That's a good point, I think Off/On might be a better way to go about it, like you say it does make it more clear what happens, while simplifying the code!

I will look into the match and see if I can make it pronounce at each correct match.